### DISCUSSION TITLE
Tarvitaanko pariakymmentä oikolukuun?

### DISCUSSION THREAD

**Discussion by ahomansikka** [voikko/discussion#10] [General]
Tarvitaanko oikoluvussa parikymmentä-sanaa, jonka molemmat osat taipuvat?

Pariakymmentä, parillekymmenelle, ...

**Comment by tlikonen** [comment]
> Tarvitaanko oikoluvussa parikymmentä-sanaa, jonka molemmat osat taipuvat?
> 
> Pariakymmentä, parillekymmenelle, ...

Tarvitaan, sanoo Kielitoimiston sanakirja:
https://www.kielitoimistonsanakirja.fi/#/parikymment%C3%A4?searchMode=all

**Comment by ahomansikka** [comment]
Lisätty.

**Comment by ahomansikka** [comment]
Lisäsin parinkymmenen lukusanana, mutta olisiko syytä muuttaa se nimisanaksi?

**Reply by yucca42** [reply]
Ilmeisesti tämäntyyppisten sanojen käsittely on jotenkin erikoista, koska Joukahaisessa ei ole sanoja parikymmentä, parisataa, parituhatta, vaikka Oikofixin analyyseissa esiintyvät sanat parikymmentä ja parisataa. Ne siis eivät analysoidu yhdyssanoiksi, joten mistä ne tulevat.

Sana ”parituhatta” analysoituu väärin, sanan ”parituhka” abessiiviksi, ja sen taivutusmuodot eivät analysoidu lainkaan. Se kai pitäisi lisätä hoitumaan samaan tapaan kuin parisataa.

**Reply by ahomansikka** [reply]
Jälleen kerran (-: Voikko ei käsittele sanoja, vaan merkkijonoja. Merkkijono "parituhatta" muutetaan aivan oiken merkkijonoksi "parituhka", sillä se on yhdistelmä merkkjonoista "pari" ja "tuhatta" , ja merkkijono "tuhatta" muutetaan aivan oiken merkkijonoksi "tuhka", joka kuvaa tuhatta-sanan perusmuotoa.

Merkkijonoja, jotka kuvaavat sanoja, joiden molemmat osat taipuvat, hoidellaan Voikossa niin, että jokaista taivutusmuotoa kuvaava merkkijono laitetaan koodiin erikseen.

**Comment by tlikonen** [comment]
* 2023-12-17 12:02:14-0800, ahomansikka wrote:
> Lisäsin parinkymmenen lukusanana, mutta olisiko syytä muuttaa se
> nimisanaksi?

Numeraali se on kielen kannalta, kuten "kolmisenkymmentä" ja muut
vastaavat likimääräiset luvut. Onko jokin tekninen syy luokitella se
substantiiviksi?

**Comment by ahomansikka** [comment]
Lukusana se on minunkin mielestäni, mutta Voikossa "parisataa" (parillasadalla jne) on nimisanana, ilmisesti teknisistä syistä, koska tällöin tunnistetaan parisataa-alkuisia yhdyssanoja. Ja sivuvaikutuksena myös omistusliitteet: parillesadalleni, ...
